src/stepFunctions/commands/visualizeStateMachine/aslVisualizationManager.ts

```diff
@@ -3,23 +3,19 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-import * as nls from 'vscode-nls'
-const localize = nls.loadMessageBundle()
 import * as vscode from 'vscode'
-import { getLogger, Logger } from '../../../shared/logger'
-import { StateMachineGraphCache } from '../../utils'
+
+import { AbstractAslVisualizationManager } from './abstractAslVisualizationManager'
 import { AslVisualization } from './aslVisualization'
+import { StateMachineGraphCache } from '../../utils'
+import { getLogger, Logger } from '../../../shared/logger'
 
-export class AslVisualizationManager {
+export class AslVisualizationManager extends AbstractAslVisualizationManager {
```

Comment:
You're not extending this class which means all `protected` can be rewritten to `private`.